### ISSUE TITLE
helm instal gpu-operator was in Init stage for a long time

### ISSUE THREAD

**Issue by JShuang7711** [NVIDIA/gpu-operator#980]
gpu-operator   nvidia-driver-daemonset-5qcw4                                     0/1     Init:0/1    0          173m
$ kubectl describe pod nvidia-driver-daemonset-5qcw4 -n gpu-operator
 subscription-config-0:
    Type:          HostPath (bare host directory volume)
    Path:          /etc/pki/entitlement
    HostPathType:  Directory
  subscription-config-1:
    Type:          HostPath (bare host directory volume)
    Path:          **/etc/yum.repos.d/redhat.repo**
    HostPathType:  File
  subscription-config-2:
    Type:          HostPath (bare host directory volume)
    Path:          /etc/rhsm
    HostPathType:  Directory
  kube-api-access-f6dtc:
    Type:                    Projected (a volume that contains injected data from multiple sources)
    TokenExpirationSeconds:  3607
    ConfigMapName:           kube-root-ca.crt
    ConfigMapOptional:       <nil>
    DownwardAPI:             true
QoS Class:                   BestEffort
Node-Selectors:              nvidia.com/gpu.deploy.driver=true
Tolerations:                 node.kubernetes.io/disk-pressure:NoSchedule op=Exists
                             node.kubernetes.io/memory-pressure:NoSchedule op=Exists
                             node.kubernetes.io/not-ready:NoExecute op=Exists
                             node.kubernetes.io/pid-pressure:NoSchedule op=Exists
                             node.kubernetes.io/unreachable:NoExecute op=Exists
                             node.kubernetes.io/unschedulable:NoSchedule op=Exists
                             nvidia.com/gpu:NoSchedule op=Exists
Events:
  Type     Reason       Age                    From     Message
  ----     ------       ----                   ----     -------
  **Warning  FailedMount  2m32s (x94 over 177m)  kubelet  MountVolume.SetUp failed for volume "subscription-config-1" : hostPath type check failed: /etc/yum.repos.d/redhat.repo is not a file**

**Comment by gaurav-cloudera** [comment]
we are also hitting the same issue while deploying the nvidia gpu operator. I see the code we have a map based on the os type and we are iterating through it to configure the mount points.

https://github.com/NVIDIA/gpu-operator/blob/dfbb6c5c4464723f6f1e4c736d3e9c3e2660deed/internal/state/driver_volumes.go#L59

I am not sure if the file is expected to be available with the OS installation or not. Can anyone please help on this.

Helm command used:
`helm install --wait --generate-name -n gpu-operator --create-namespace nvidia/gpu-operator --version=v25.3.0 --set mig.strategy=single`

`Events:
  Type     Reason       Age                 From      Message
  ----     ------       ----                ----      -------
  Normal   Scheduled    114s                yunikorn  Successfully assigned gpu-operator/nvidia-driver-daemonset-sq5d7 to <node>
  Warning  FailedMount  50s (x8 over 114s)  kubelet   MountVolume.SetUp failed for volume "subscription-config-1" : hostPath type check failed: /etc/yum.repos.d/redhat.repo is not a file
gmalik@Gauravs-MacBook-Pro default %`

**Comment by gaurav-cloudera** [comment]
@shivamerla could you please share any pointers on this.

**Comment by cdesiniotis** [comment]
@JShuang7711 can you please provide more information about your cluster (k8s, container runtime, etc.) and how you are installing GPU Operator (version, values, etc.)?

**Comment by rahulait** [comment]
This issue has been open for over 90 days without recent updates, and the context may now be outdated.

To keep the issue tracker clean and focused on current and actionable topics, I am going to close this issue as we haven't received logs as requested before. If you are still blocked by this or have more queries, please feel free to re-open it with requested logs or create a new issue so that someone from team can take a look.
